.github/CODEOWNERS

```diff
@@ -0,0 +1,37 @@
+# This is a comment.
```

Comment:
Is it usual to add this kind of stuff to forked code?

Reply:
First time doing this so not sure what is typical, but my thought process with this PR was:

We will be maintaining this version of the package. We need folks from our team to review PRs. If someone uses the pypi package we publish and finds and issue, they should contact us about it (it is possible the issue will apply to both versions of the package, but we certainly don't want anyone reaching out to the original author about issues that are specific to our fork, right?). So I think these updates are probably good/right, but we should make sure to clearly credit the original author in other locations.

Reply:
This seems OK to me